### ISSUE TITLE
For accessibility, the label's for attribute needs to hook up to the input's id attribute.

### ISSUE THREAD

**Issue by lmcglohon** [archivesspace/archivesspace#1334]

**Comment by cfitz** [comment]
I wish I knew more about accessibility..but one thing I do know is that label tags for attirbutes should point to a inputs id attribute, even if the label is not visible. 

<label for="myinput">Look! An input:</label>
<input type='text' name='whatever' id='myinput'>

This is an issue all through out Aspace, but I noticed that #1290 removed a bunch of id's, which is logging some errors ( #1331). But also, it removes the input id as well. Hence, this ticket.
